scripts/jenkins/ardana/ansible/roles/crowbar_register/tasks/reserve_admin_ip.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+- name: Get crowbar admin_network databag contents
```

Comment:
Would be really nice if we had an ansible python module doing this kind of thing, that we could reuse in other tasks as well.